projects/solutions/zihe/ConnectFour/src/Game.js

```diff
@@ -0,0 +1,173 @@
+import React, { useEffect, useState } from "react";
+import styled from 'styled-components';
+import Gameboard from './components/Gameboard';
+import { checkWinner } from './helpers/checkWinner';
+import { checkRow } from "./helpers/checkRow";
+import { MAX_COL } from "./constants";
+
+const GameWrapper = styled.div`
+width:100vw;
+height: 100vh;
+background-color: black;
+display: flex;
+justify-content: center;
+align-items: center;
+flex-direction: column;
+color: white;
+`
+const GameInfo = styled.div`
+display: flex;
+justify-content: center;
+align-items: center;
+flex-direction: column;
+`
+const UserMark = styled.div`
+width:0px;
+height:0px;
+border: 25px solid goldenrod;
+border-radius: 50%;
+`
+const ComputerMark = styled.div`
+width:0px;
+height:0px;
+border: 25px solid silver;
+border-radius: 50%;
+`
+const ColorIndicator = styled.div`
+display: flex;
+justify-content: center;
+align-items: center;
+`
+const Marker = styled.span`
+display:flex;
+justify-content: center;
+align-items: center;
+padding-left: 20px;
+`
+const RestartButton = styled.button`
+margin:10px;
+border-radius: 10px;
+width: 100px;
+height: 40px;
+border: 1px solid;
+:hover {
+  cursor: pointer;
+}
+`
+const Game = () => {
+  const [checkWinnerRow, setCheckWinnerRow] = useState(0);
+  const [checkWinnerCol, setCheckWinnerCol] = useState(0);
```

Comment:
这样没问题, 但是我觉得他们可以放一起 - 同一个variable
因为你每次都是setCheckWinnerRow 和setCheckWinnerCol 同时使用, 那他们2个合并一下会更好
```suggestion
  const [coordinate, setCoordinate] = useState([0,0]);
```